src/s25update.cpp

```diff
@@ -22,6 +22,11 @@
 #    include <shellapi.h>
 #endif
 
+// taken from curl/curlver.h of libCURL 7.43+ for easier readability.
+#ifndef CURL_AT_LEAST_VERSION
+#    define CURL_AT_LEAST_VERSION(x, y, z) (LIBCURL_VERSION_NUM >= ((x) << 16 | (y) << 8 | (z)))
+#endif
```

Comment:
I meant to also swap this 2 lines (you can just accept this in the Github UI if you agree)

```suggestion
#ifndef CURL_AT_LEAST_VERSION
// taken from curl/curlver.h of libCURL 7.43+ for easier readability.
#    define CURL_AT_LEAST_VERSION(x, y, z) (LIBCURL_VERSION_NUM >= ((x) << 16 | (y) << 8 | (z)))
#endif
```

Reply:
Whoops... did mess up and fixed my mess. Also included the suggestion.

Reply:
No worries. Can you test that this (still) works for you or did you already?

Reply:
Yes i did test it, both at runtime and with make test and it looks good to me.